src/views/anon/intro/WhatWeOffer.tsx

```diff
@@ -0,0 +1,192 @@
+                        What we offer
+                    </Typography>
+                    <Typography sx={{mt: -25, display: {lg: 'none', xl: 'flex'}, marginLeft: '30rem', fontWeight: '600'}} variant='h2' component='div' gutterBottom>
+                        What we offer
+                    </Typography>
+
+                    <Typography sx={{display: 'flex', fontSize: '110%', alignItems: 'center', justifyContent: 'center', marginLeft: '30rem', paddingBottom: '30px'}} align="center" variant='subtitle1' component='div' gutterBottom>
+                    At Venture Build, we offer a dynamic ecosystem of features designed to fuel the<br/> aspirations of young African innovators. Discover the tools that empower you to<br/>
+                     thrive and make your mark in the startup landscape.                    
+                     </Typography>
+
+                    <Box sx={{display: 'flex', justifyContent: 'center', width: '100%', alignItems: 'space-evenly', marginLeft: '30rem'}}>
+                        <Button size='large' variant='outlined' sx={{color: '#000'}}>Demo</Button>
+                        <Link to={PathConstants.signUp}>
+                            <Button size='large' variant='contained' sx={{color: '#fff', alignItems: 'left', marginLeft: '10px', backgroundColor: '#DC6803', '&:hover': { backgroundColor: '#E8822A'}}}>Sign up</Button>
+                        </Link>
+                    </Box>
+                    </Grid>
+            </Box>
+            <Box sx={{mt: -40, display: { lg: 'flex' }, height: '80vh', marginLeft: '20px', width: '183vh', alignItems: 'center', justifyContent: 'center', overflowX: 'hidden', backgroundImage: `url(${home})`, backgroundRepeat: "no-repeat",
+    backgroundSize: "cover", backgroundPosition: `right bottom`}}>
+
+            </Box>
+
+            <Box sx={{paddingTop: '40px', paddingBottom: '40px', backgroundImage: 'linear-gradient(to bottom, #F2F2F2 60%, #FFFFFF 40%, rgba(203, 65, 100, 0))'}}>
+                <Grid sx={{width: '100%', height: '100%'}} container spacing={0}>
+                    <Grid item lg={6} sx={{height: '100%', width: '100%', padding: '0 80px 20px 80px', display: 'flex !important', alignItems: 'start !important', justifyContent: 'start !important', flexDirection: 'column !important'}}>
+                        <Typography variant='overline' fontWeight='bold' color='#B54708'>
+                            Overview
+                        </Typography>
+                        <Typography sx={{display: {xs: 'inline', xl: 'none'}}} variant='h3' gutterBottom>
+                        Beyond a co-working space
+                        </Typography>
+                        <Typography sx={{display: {xs: 'none', xl: 'inline'}}} variant='h3' gutterBottom>
+                        Beyond a co-working space
+                        </Typography>
```

Comment:
These have exact same content and styling, so I don't think we need to vary over display sizes